packages/react-native-bpk-component-banner-alert/src/BpkBannerAlert.android.js

```diff
@@ -132,9 +135,11 @@ const BpkBannerAlert = (props: Props) => {
   const rowContent = (
     <Fragment>
       <BpkIcon style={[STYLES.icon, iconStyle]} icon={icon} small />
-      <BpkText style={STYLES.message} textStyle="sm">
-        {message}
-      </BpkText>
+      <View style={STYLES.messageContainer}>
```

Comment:
Would it be possible to add a small comment explaining the need for this fix, just to prevent it accidentally regressing in the future. Then I think this is 💯

Reply:
@georgegillams Done 👍 Let me know if you're happy with that.